packages/frontend/src/components/DashboardTiles/TileBase/index.tsx

```diff
@@ -199,7 +199,13 @@ const TileBase = <T extends Dashboard['tiles'][number]>({
                         tile={tile}
                         onClose={() => setIsReplacingChart(false)}
                         onConfirm={(data) => {
-                            onEdit(data);
+                            onEdit({
+                                ...data,
+                                properties: {
+                                    ...data.properties,
+                                    title: undefined,
```

Comment:
`TileUpdateModal` is used for all tile types. This would remove the title from any change in the loom/markdown tile

Reply:
This fix might be easier after the changes from this PR https://github.com/lightdash/lightdash/pull/6281